<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-24H1DPQU" name="Default test" tests="2" failures="0" timestamp="22 Oct 2016 12:45:07 GMT" time="49.317" errors="0">
  <testcase name="@BeforeClass configure" time="0.0" classname="Testcases.LoginTest">
    <failure type="java.lang.ExceptionInInitializerError">
      <![CDATA[java.lang.ExceptionInInitializerError
	at Pages.loginpage.<init>(loginpage.java:19)
	at Testcases.LoginTest.configure(LoginTest.java:53)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:510)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:211)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:170)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:104)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
Caused by: org.openqa.selenium.WebDriverException: Failed to connect to binary FirefoxBinary(C:\Program Files (x86)\Mozilla Firefox\firefox.exe) on port 7055; process output follows: 
ker for PreviousExperimentProvider
1477140263527	addons.manager	DEBUG	Provider finished startup: PreviousExperimentProvider
1477140263533	DeferredSave.extensions.json	WARN	Write failed: Win error 32 during operation move on file C:\Users\NEHAAN~1\AppData\Local\Temp\anonymous3897136757464313814webdriver-profile\extensions.json.tmp (The process cannot access the file because it is being used by another process.
) ((unknown module)) No traceback available
1477140263534	addons.xpi-utils	WARN	Failed to save XPI database: Win error 32 during operation move on file C:\Users\NEHAAN~1\AppData\Local\Temp\anonymous3897136757464313814webdriver-profile\extensions.json.tmp (The process cannot access the file because it is being used by another process.
) ((unknown module)) No traceback available
1477140263535	addons.xpi-utils	WARN	Failed to save XPI database: Win error 32 during operation move on file C:\Users\NEHAAN~1\AppData\Local\Temp\anonymous3897136757464313814webdriver-profile\extensions.json.tmp (The process cannot access the file because it is being used by another process.
) ((unknown module)) No traceback available
1477140263535	addons.xpi-utils	WARN	Failed to save XPI database: Win error 32 during operation move on file C:\Users\NEHAAN~1\AppData\Local\Temp\anonymous3897136757464313814webdriver-profile\extensions.json.tmp (The process cannot access the file because it is being used by another process.
) ((unknown module)) No traceback available
1477140263536	addons.xpi-utils	WARN	Failed to save XPI database: Win error 32 during operation move on file C:\Users\NEHAAN~1\AppData\Local\Temp\anonymous3897136757464313814webdriver-profile\extensions.json.tmp (The process cannot access the file because it is being used by another process.
) ((unknown module)) No traceback available
1477140263538	DeferredSave.addons.json	DEBUG	Starting write
1477140263885	DeferredSave.addons.json	DEBUG	Write succeeded
JavaScript error: , line 0: uncaught exception: [object Object]
JavaScript error: chrome://fxdriver/content/server.js, line 61: NS_ERROR_XPC_CI_RETURNED_FAILURE: Component returned failure code: 0x80570015 (NS_ERROR_XPC_CI_RETURNED_FAILURE) [nsIJSCID.createInstance]
1477140279448	addons.manager	DEBUG	shutdown
1477140279449	addons.manager	DEBUG	Calling shutdown blocker for XPIProvider
1477140279450	addons.xpi	DEBUG	shutdown
1477140279450	addons.xpi-utils	DEBUG	shutdown
1477140279451	addons.xpi-utils	WARN	Error: XPI database modified after shutdown began (resource://gre/modules/addons/XPIProvider.jsm -> resource://gre/modules/addons/XPIProviderUtils.js:428:16) JS Stack trace: this.XPIDatabase.saveChanges@XPIProviderUtils.js:428:17 < XPIDB_shutdown@XPIProviderUtils.js:1009:9 < XPI_shutdown@XPIProvider.jsm:2451:18 < _startProvider/AMProviderShutdown/<@AddonManager.jsm:718:21 < Promise@Promise-backend.js:351:5 < _startProvider/AMProviderShutdown@AddonManager.jsm:716:16 < Barrier/this.client.addBlocker/promise</trigger@AsyncShutdown.jsm:652:23 < Barrier.prototype<._wait@AsyncShutdown.jsm:790:7 < Barrier.prototype<.wait@AsyncShutdown.jsm:774:28 < AddonManagerInternal.shutdownManager<@AddonManager.jsm:1108:15
1477140279454	DeferredSave.extensions.json	DEBUG	Save changes
1477140279455	DeferredSave.extensions.json	DEBUG	Flush called while data is dirty
1477140279458	addons.manager	DEBUG	Calling shutdown blocker for LightweightThemeManager
1477140279459	addons.manager	DEBUG	Calling shutdown blocker for GMPProvider
1477140279463	addons.manager	DEBUG	Calling shutdown blocker for PluginProvider
1477140279464	addons.manager	DEBUG	Calling shutdown blocker for <unnamed-provider>
1477140279466	addons.manager	DEBUG	Calling shutdown blocker for PreviousExperimentProvider
1477140279471	DeferredSave.extensions.json	DEBUG	Starting write
1477140279500	DeferredSave.extensions.json	DEBUG	Write succeeded
1477140279500	addons.xpi-utils	DEBUG	XPI Database saved, setting schema version preference to 17
1477140279500	addons.xpi	DEBUG	Notifying XPI shutdown observers
1477140279502	addons.manager	DEBUG	Async provider shutdown done

Build info: version: '2.53.0', revision: '35ae25b', time: '2016-03-15 16:57:40'
System info: host: 'LAPTOP-24H1DPQU', ip: '192.168.1.40', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: driver.version: Driver
	at org.openqa.selenium.firefox.internal.NewProfileExtensionConnection.start(NewProfileExtensionConnection.java:124)
	at org.openqa.selenium.firefox.FirefoxDriver.startClient(FirefoxDriver.java:271)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:119)
	at org.openqa.selenium.firefox.FirefoxDriver.<init>(FirefoxDriver.java:218)
	at org.openqa.selenium.firefox.FirefoxDriver.<init>(FirefoxDriver.java:211)
	at org.openqa.selenium.firefox.FirefoxDriver.<init>(FirefoxDriver.java:207)
	at org.openqa.selenium.firefox.FirefoxDriver.<init>(FirefoxDriver.java:120)
	at UtilityFiles.Driver.<clinit>(Driver.java:16)
	... 26 more
Caused by: org.openqa.selenium.firefox.NotConnectedException: Unable to connect to host 127.0.0.1 on port 7055 after 45000 ms. Firefox console output:
ker for PreviousExperimentProvider
1477140263527	addons.manager	DEBUG	Provider finished startup: PreviousExperimentProvider
1477140263533	DeferredSave.extensions.json	WARN	Write failed: Win error 32 during operation move on file C:\Users\NEHAAN~1\AppData\Local\Temp\anonymous3897136757464313814webdriver-profile\extensions.json.tmp (The process cannot access the file because it is being used by another process.
) ((unknown module)) No traceback available
1477140263534	addons.xpi-utils	WARN	Failed to save XPI database: Win error 32 during operation move on file C:\Users\NEHAAN~1\AppData\Local\Temp\anonymous3897136757464313814webdriver-profile\extensions.json.tmp (The process cannot access the file because it is being used by another process.
) ((unknown module)) No traceback available
1477140263535	addons.xpi-utils	WARN	Failed to save XPI database: Win error 32 during operation move on file C:\Users\NEHAAN~1\AppData\Local\Temp\anonymous3897136757464313814webdriver-profile\extensions.json.tmp (The process cannot access the file because it is being used by another process.
) ((unknown module)) No traceback available
1477140263535	addons.xpi-utils	WARN	Failed to save XPI database: Win error 32 during operation move on file C:\Users\NEHAAN~1\AppData\Local\Temp\anonymous3897136757464313814webdriver-profile\extensions.json.tmp (The process cannot access the file because it is being used by another process.
) ((unknown module)) No traceback available
1477140263536	addons.xpi-utils	WARN	Failed to save XPI database: Win error 32 during operation move on file C:\Users\NEHAAN~1\AppData\Local\Temp\anonymous3897136757464313814webdriver-profile\extensions.json.tmp (The process cannot access the file because it is being used by another process.
) ((unknown module)) No traceback available
1477140263538	DeferredSave.addons.json	DEBUG	Starting write
1477140263885	DeferredSave.addons.json	DEBUG	Write succeeded
JavaScript error: , line 0: uncaught exception: [object Object]
JavaScript error: chrome://fxdriver/content/server.js, line 61: NS_ERROR_XPC_CI_RETURNED_FAILURE: Component returned failure code: 0x80570015 (NS_ERROR_XPC_CI_RETURNED_FAILURE) [nsIJSCID.createInstance]
1477140279448	addons.manager	DEBUG	shutdown
1477140279449	addons.manager	DEBUG	Calling shutdown blocker for XPIProvider
1477140279450	addons.xpi	DEBUG	shutdown
1477140279450	addons.xpi-utils	DEBUG	shutdown
1477140279451	addons.xpi-utils	WARN	Error: XPI database modified after shutdown began (resource://gre/modules/addons/XPIProvider.jsm -> resource://gre/modules/addons/XPIProviderUtils.js:428:16) JS Stack trace: this.XPIDatabase.saveChanges@XPIProviderUtils.js:428:17 < XPIDB_shutdown@XPIProviderUtils.js:1009:9 < XPI_shutdown@XPIProvider.jsm:2451:18 < _startProvider/AMProviderShutdown/<@AddonManager.jsm:718:21 < Promise@Promise-backend.js:351:5 < _startProvider/AMProviderShutdown@AddonManager.jsm:716:16 < Barrier/this.client.addBlocker/promise</trigger@AsyncShutdown.jsm:652:23 < Barrier.prototype<._wait@AsyncShutdown.jsm:790:7 < Barrier.prototype<.wait@AsyncShutdown.jsm:774:28 < AddonManagerInternal.shutdownManager<@AddonManager.jsm:1108:15
1477140279454	DeferredSave.extensions.json	DEBUG	Save changes
1477140279455	DeferredSave.extensions.json	DEBUG	Flush called while data is dirty
1477140279458	addons.manager	DEBUG	Calling shutdown blocker for LightweightThemeManager
1477140279459	addons.manager	DEBUG	Calling shutdown blocker for GMPProvider
1477140279463	addons.manager	DEBUG	Calling shutdown blocker for PluginProvider
1477140279464	addons.manager	DEBUG	Calling shutdown blocker for <unnamed-provider>
1477140279466	addons.manager	DEBUG	Calling shutdown blocker for PreviousExperimentProvider
1477140279471	DeferredSave.extensions.json	DEBUG	Starting write
1477140279500	DeferredSave.extensions.json	DEBUG	Write succeeded
1477140279500	addons.xpi-utils	DEBUG	XPI Database saved, setting schema version preference to 17
1477140279500	addons.xpi	DEBUG	Notifying XPI shutdown observers
1477140279502	addons.manager	DEBUG	Async provider shutdown done

	at org.openqa.selenium.firefox.internal.NewProfileExtensionConnection.start(NewProfileExtensionConnection.java:112)
	... 33 more
]]>
    </failure>
  </testcase> <!-- @BeforeClass configure -->
  <testcase name="@BeforeMethod startlogin" time="0.0" classname="Testcases.LoginTest">
    <skipped/>
  </testcase> <!-- @BeforeMethod startlogin -->
  <testcase name="@BeforeMethod startlogin" time="0.0" classname="Testcases.LoginTest">
    <skipped/>
  </testcase> <!-- @BeforeMethod startlogin -->
  <testcase name="ValidLoginToSonosm" time="0.007" classname="Testcases.LoginTest">
    <skipped/>
  </testcase> <!-- ValidLoginToSonosm -->
  <testcase name="InvalidloginToSonosim" time="0.02" classname="Testcases.LoginTest">
    <skipped/>
  </testcase> <!-- InvalidloginToSonosim -->
</testsuite> <!-- Default test -->
